Solitaire/js/main.js

```diff
@@ -1,4 +1,259 @@
-
+(function () {
+    var $btnReset = document.getElementById('reset');
+    var cards = [];
+	var playingDecks = {
+		deckPlaying1: {array: [], cardPos: 1, click: checkCard},
+		deckPlaying2: {array: [], cardPos: 3, click: checkCard},
+		deckPlaying3: {array: [], cardPos: 6, click: checkCard},
+		deckPlaying4: {array: [], cardPos: 10, click: checkCard},
+		deckPlaying5: {array: [], cardPos: 15, click: checkCard},
+		deckPlaying6: {array: [], cardPos: 21, click: checkCard},
+		deckPlaying7: {array: [], cardPos: 28, click: checkCard},
+		deckDistribution: {array: [], cardPos: 52, click: openCardDestribution},
+		deckDistributionOpen: {array: [], click: chooseCardDistribution},
+		deckFinish1: {array: [], click: checkCard},
+		deckFinish2: {array: [], click: checkCard},
+		deckFinish3: {array: [], click: checkCard},
+		deckFinish4: {array: [], click: checkCard}
+	}
+	var mixRandom = function() {
+        return Math.random() - 0.5;
+    };
+	var deckSelected = null;
+    var cardSelectedNumber = null;
+
+    
+    function fillingArrayCards() {
```

Comment:
Bad naming. The function name that does some actions or logic should starts with a verb. In this case **fillArrayCards**